### PR TITLE
Added permissions documentation

### PR DESCRIPTION
**Pull request by jasonf20**

### CODE REVIEW

**Comment by jasonf20** [comment]
@shanielh If you want to generate the permissions list documents I won't mind :P

**Comment by shanielh** [comment]
@Jason would you like to add some links to those pages in `SUMMARY.md`?

**Comment by jasonf20** [comment]
I would have done it then if I remembered how to add links in markdown

**Comment by shanielh** [comment]
Hint: [This file is full of links](https://github.com/Upsolver/Documentation/blob/master/SUMMARY.md)

File: Permissions/Overview.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # Permissions Overview
 
-Upsolver supports IAM (Identity Access Management) style permissions. This means you can create users, groups, policies and relationships between them to define the authorized operations allowed foe each user.
```
Comment (shanielh) [inline Permissions/Overview.md]: IAM - Identity & Access Management

**Pull request merged**